### DISCUSSION TITLE
Customize search border

### DISCUSSION THREAD

**Discussion by dennysutanto** [Power-Components/livewire-powergrid#1569] [Themes]
Hi I'm new to this plugin, i have explored the documentation but i cannot find answers.
How to change the border of the search box, also how to remove the double arrow as shown in the screen shot below ?

Thanks

![image](https://github.com/Power-Components/livewire-powergrid/assets/2149385/8bc35851-8e8e-4318-9bca-c036a3347df8)

**Comment by tegarmonster** [comment]
Hi @dennysutanto ,
You can apply a custom theme to apply the desired changes.
ref: https://livewire-powergrid.com/expanding-powergrid/custom-theme.html
